wordpress_connector/models/contracts.py

```diff
@@ -234,6 +234,8 @@ def create_sponsor_from_web(self):
         })
         self.child_id.sponsor_id = partner
 
+        partner.set_privacy_statement(origin='Website partner creation')
```

Comment:
There is the case of a new sponsorship that is missing. Here it will do it only if it's a new partner, but what about when an existing partner makes a new sponsorship?